pkg/universe.dagger.io/python/python.cue

```diff
@@ -2,28 +2,60 @@
 package python
 
 import (
-	"universe.dagger.io/docker"
+	"dagger.io/dagger"
+	"dagger.io/dagger/core"
 
+	"universe.dagger.io/docker"
 	"universe.dagger.io/alpine"
 )
 
 // Run a python script in a container
 #Run: {
 	// Contents of the python script
-	script: string
+	script: {
+		// A directory containing one or more bash scripts
+		directory: dagger.#FS
+
+		// Name of the file to execute
+		filename: string
+
+		_directory: directory
+		_filename:  filename
+	} | {
+		// Script contents
+		contents: string
+
+		_filename: "run.py"
+		_write:    core.#WriteFile & {
+			input:      dagger.#Scratch
+			path:       _filename
+			"contents": contents
+		}
+		_directory: _write.output
+	}
+
+	// arguments to the script
+	args: [...string]
+
+	// where to mount the script inside the container
+	_mountpoint: "/run/python"
 
-	// FIXME: don't pass the script as argument: write to filesystme instead
 	docker.#Run & {
 		command: {
-			name: "python"
-			flags: "-c": script
+			name:   "python3"
+			"args": ["\(_mountpoint)/\(script._filename)"] + args
 		}
 
 		// As a convenience, image defaults to a ready-to-use python environment
-		image: docker.#Image | *_defaultImage
+		input: docker.#Image | *_defaultImage.output
 
-		_defaultImage: alpine.#Image & {
+		_defaultImage: alpine.#Build & {
```

Comment:
Yeah I could do that if it is preferred. The existing `python` package uses this method -- I only changed the alpine method.

It might be worthwhile to do that separately? This is a fix, that would be a refactor.

Reply:
This way is fine. We can refactor further later.